src/LookupTable/components/ImportCsvModal.tsx

```diff
@@ -123,18 +136,24 @@ const ImportCsvModal: React.FC<any> = ({
           />
         ) : (
           <Form>
-            {!tableLabel && (
-              <Form.Field>
-                <label>{t('LOOKUP_TABLE_NAME')}</label>
-                <input
-                  placeholder={t('LOOKUP_TABLE_NAME')}
-                  value={name || ''}
-                  onChange={(event) =>
-                    dispatch({ type: 'SET_TABLE_NAME', payload: event.target.value })
-                  }
-                />
-              </Form.Field>
-            )}
+            <Form.Field>
+              <label>{t('LOOKUP_TABLE_NAME')}</label>
+              <input
+                placeholder={t('LOOKUP_TABLE_NAME')}
+                value={name || ''}
+                onChange={(event) =>
+                  dispatch({ type: 'SET_TABLE_NAME', payload: event.target.value })
+                }
+              />
+            </Form.Field>
```

Comment:
This field is no longer conditional -- I'm allowing the Display Name to be edited on table update (doesn't change the underlying table name)